dallinger/deployment.py

```diff
@@ -575,11 +579,9 @@ def deploy_sandbox_shared_setup(log, verbose=True, app=None, exp_config=None):
     log("Experiment details:")
     log("App home: {}".format(result["app_home"]), chevrons=False)
     log("Dashboard URL: {}".format(result["dashboard_url"]), chevrons=False)
+    log("Dashboard user: {}".format(config.get("dashboard_user")), chevrons=False)
     log(
-        "Dashboard user: {}".format(heroku_config.get("DASHBOARD_USER")), chevrons=False
-    )
-    log(
-        "Dashboard password: {}".format(heroku_config.get("DASHBOARD_PASSWORD")),
+        "Dashboard password: {}".format(config.get("dashboard_password")),
```

Comment:
Then we'd need to record whether the password was auto-generated (default) or came from an existing config. I don't think there's any real security advantage to to doing that, and it will add code complexity.